README.md

```diff
@@ -3,10 +3,25 @@
 [![CircleCI](https://circleci.com/gh/solidusio/solidus_braintree.svg?style=shield)](https://circleci.com/gh/solidusio/solidus_braintree)
 [![codecov](https://codecov.io/gh/solidusio/solidus_braintree/branch/master/graph/badge.svg)](https://codecov.io/gh/solidusio/solidus_braintree)
 
-`solidus_braintree` is an extension that adds support for using [Braintree](https://www.braintreepayments.com) as a payment source in your [Solidus](https://solidus.io/) store. It supports Apple Pay, PayPal, and credit card transactions.
+`solidus_braintree` is an extension that adds support for using [Braintree](https://www.braintreepayments.com) as a payment source in your [Solidus](https://solidus.io/) store. It supports Apple Pay, PayPal, Venmo, and credit card transactions. The extension uses Braintree's [JavaScript v3 SDK](https://braintree.github.io/braintree-web/current/).
 
 🚧 This extension is currently only compatible with the `solidus_starter_frontend` 🚧
```

Comment:
Maybe we can remove this line and add a "Compatibility" section, where we explain which extension version is compatible with which Solidus version?